### ISSUE TITLE
[Screen reader - ADF - Azure data factory] Screen reader doesn't notify transferred blob containers on Toggle explorer window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3853]
**User Experience:**
Screen reader user will be affected if screen reader doesn't notify user about about any notification. Not only this, user won't get an idea of such notification.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1
Screen reader: Narrator, NVDA

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it  

**Repro Steps:​**
- Launch Azure Storage Explorer app.
- Navigate and select 'Toggle Explorer' at left panel.
- Now, navigate to any of the blob containers and copy that blob containers to Paste it over another blob containers.
- Verify copied blob containers by using screen reader at time of paste.

**Actual Result:**
Screen reader doesn't notify transferred blob containers on Toggle Explorer window. It only announces 'Alert 1 activity added user arrow key to navigate the application. It doesn't announce which blob container is being transferred. 

**Expected Result:**
Screen reader must notify about blob container transfer on Toggle Explorer window. It should  announce 'Alert 1 activity added along with name of blob container so that screen reader user could get an idea that which blob container has been transferred.

**Note:**
- NVDA also doesn't notify about transferred Blob containers.
- This issue also exists on MAC with voice-over.

[MAS4.1.3_Toggle Explorer.zip](https://github.com/microsoft/AzureStorageExplorer/files/5641830/MAS4.1.3_Toggle.Explorer.zip)
